versioned/tiered/rocksdb/src/main/java/com/dremio/nessie/versioned/store/rocksdb/Condition.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.dremio.nessie.versioned.store.rocksdb;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A set of conditions that are asserted against an Entity.
+ */
+public class Condition {
+
+  public List<Function> functionList;
```

Comment:
Should never have public members - or as Laurent already suggested we can use Immutables.

Reply:
I updated the member to make this private then added a getting to return an immutable copy.

Reply:
Can you look at using Immutables (https://immutables.github.io/) instead?

Reply:
Will do

Reply:
Done